docs/docs/guide/nextjs-13-and-growthbook.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: GrowthBook and Next.js 13 (beta)
+description: This guide walks you through using GrowthBook with Next.js 13 (beta).
+sidebar_label: Next.js 13 (beta)
+slug: nextjs-13-and-growthbook
+---
+
+# GrowthBook and Next.js 13 (beta)
+
+This document is a guide on how to add GrowthBook feature flags to your Next.js application. It assumes you are starting from scratch, and are using the new 'app directory' pattern introduced in Next 13.
+
+If you already have a NextJS application, you can skip to step 2.
```

Comment:
Is it NextJS or Next.js? We should probably stick to whatever their branding guidelines are with regards to their naming.

Reply:
Seems like it's Next.js - updated